app/iht/forms/registration/DeceasedForms.scala

```diff
@@ -63,7 +63,11 @@ trait DeceasedForms extends IhtFormValidator {
         .verifying("error.lastName.giveUsingXCharsOrLess", f => f.length <= appConfig.validationMaxLengthLastName)
         .verifying("error.lastName.giveUsingOnlyValidChars", f => nameAndAddressRegex.findFirstIn(f).fold(false)(_=>true)),
       "nino" -> ninoForDeceased(
-        "error.nino.give","error.nino.giveUsing8Or9Characters","error.nino.giveUsingOnlyLettersAndNumbers", oRegDetails),
+        "error.nino.give",
+        "error.nino.giveUsing8Or9Characters",
+        "error.nino.giveUsingOnlyLettersAndNumbers",
+        oRegDetails
```

Comment:
Question - is this missing 'loginNino' as used in CoExecutor forms to enable the verify to be checked?

Reply:
`.verifying("error.nino.deceased.sameaslogin", _ != loginNino)` - Is the piece of code below that should be checking for that - I needed a specific exception for CoExecutor forms due to way that works and you've got more data entered by that point of the service. I wrote a unit test for that case. 
I may not need the `.verifying` for the CoExecutor forms, just the `loginNino` argument, so I could write a test and check if I don't need it there to get some confidence.

Reply:
Ok, seems it's needed. I'll push up an extra test, but I don't think there needs to be code changes.

Reply:
Agreed - just done some exploratory testing and looks perfectly OK meeting the criteria. Only observation that I've mentioned is the ability to pass in a nino without suffix within the app eg AB123456 - this passes verification but cannot be reused in the applicants details if already used in deceased details. I think this would be a different issue however.